rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -1505,7 +1505,19 @@ message GetRunningSnsVersionResponse {
   // The currently deployed version of the SNS.
   Governance.Version deployed_version = 1;
   // The upgrade in progress, if any.
-  Governance.UpgradeInProgress pending_version = 2;
+  UpgradeInProgress pending_version = 2;
+
+  message UpgradeInProgress {
+    // Version to  be upgraded to
+    Governance.Version target_version = 1;
+    // Seconds since UNIX epoch to mark this as a failed version if not in sync with current version
+    uint64 mark_failed_at_seconds = 2;
+    // Lock to avoid checking over and over again.  Also, it is a counter for how many times we have attempted to check,
+    // allowing us to fail in case we otherwise have gotten stuck.
+    uint64 checking_upgrade_lock = 3;
+    // The proposal that initiated this upgrade
+    uint64 proposal_id = 4;
+  }
```

Comment:
The pending_version in the governance proto has been changed to have `proposal_id` be optional. This change keeps `GetRunningSnsVersionResponse`'s API the same – if `proposal_id` were changed to be optional in the API, it would no longer be backwards-compatible (as candid cannot decode `T` if it was encoded as `opt T`).